### PR TITLE
DMU 10 Cards + 3 card fixes

### LINKED ISSUE #1457: Rith, Liberated Primeval triggering from the library
**Describe the bug**
While playing a commander in the snapshot from yesterday 1.6.54.902, Rith, Liberated Primeval triggers the eot trigger to create a 4/4 red dragon creature token with flying. This is triggering from the library instead of while it is inplay.

**Expected behavior**
Rith, Liberated should trigger while in play only.

**Screenshots**
image
image

### PR DESCRIPTION
**Pull request by Simisays**
-closes #1457

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/haunting_figment.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Haunting Figment
+ManaCost:1 U
+Types:Creature Illusion
+PT:2/1
+K:Vigilance
+S:Mode$ Continuous | Affected$ Card.Self | AddHiddenKeyword$ Unblockable | CheckSVar$ X | PresentCompare$ EQ0 | Description$ CARDNAME can't be blocked as long as you've cast an instant or sorcery spell this turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/haunting_figment.txt:6]: PresentCompare$ has no place there

File: forge-gui/res/cardsfolder/upcoming/karns_sylex.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Karn's Sylex
+ManaCost:3
+Types:Legendary Artifact
+K:CARDNAME enters the battlefield tapped.
+S:Mode$ CantPayLife | ValidPlayer$ Player | ValidCause$ Spell,Activated.nonManaAbility | ForCost$ True | Description$ Players can't pay life or sacrifice creatures to cast spells or activate abilities that aren't mana abilities.
+S:Mode$ CantSacrifice | ValidCard$ Creature | ValidCause$ Spell,Activated.nonManaAbility | ForCost$ True | Secondary$ True | Description$ Players can't pay life or sacrifice creatures to cast spells or activate abilities that aren't mana abilities.
+A:AB$ DestroyAll | Cost$ X T Exile<1/CARDNAME> | ValidCards$ Permanent.nonLand+cmcEQX | SorcerySpeed$ True | SpellDescription$ Destroy each nonland permanent with mana value X or less. Activate only as a sorcery.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/karns_sylex.txt:7]: LEX

File: forge-gui/res/cardsfolder/upcoming/najal_the_storm_runner.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Name:Najal, the Storm Runner
+ManaCost:2 U U R
+Types:Legendary Creature Efreet Wizard
+PT:5/4
+S:Mode$ CastWithFlash | ValidCard$ Sorcery | ValidSA$ Spell | Caster$ You | Description$ You may cast sorcery spells as though they had flash.
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigImmediateTrig | TriggerDescription$ Whenever CARDNAME, you may pay {2}. If you do, when you cast your next instant or sorcery spell this turn, copy it. You may choose new targets for the copy.
+SVar:TrigImmediateTrig:AB$ ImmediateTrigger | Cost$ 2 | Execute$ TrigDelayedTrigger
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/najal_the_storm_runner.txt:7]: "if you do" = normal trigger (without Immediate)
"when you do" = reflexive

File: forge-gui/res/cardsfolder/upcoming/haunting_figment.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Haunting Figment
+ManaCost:1 U
+Types:Creature Illusion
+PT:2/1
+K:Vigilance
+S:Mode$ Continuous | Affected$ Card.Self | AddHiddenKeyword$ Unblockable | CheckSVar$ X | SVarCompare$ GE1 | Description$ CARDNAME can't be blocked as long as you've cast an instant or sorcery spell this turn.
+SVar:X:Count$ThisTurnCast_Instant.YouCtrl,Sorcery.YouCtrl
+SVar:BuffedBy:Instant,Sorcery
+DeckHints:Type$Instant|Sorcery
+Oracle:Vigilance\nHaunting Figment can’t be blocked as long as you’ve cast an instant or sorcery spell this turn.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/haunting_figment.txt:10]: can’t and you’ve have curlys

File: forge-gui/res/cardsfolder/upcoming/najal_the_storm_runner.txt [hunk 5]
```diff
@@ -0,0 +1,12 @@
+Name:Najal, the Storm Runner
+ManaCost:2 U U R
+Types:Legendary Creature Efreet Wizard
+PT:5/4
+S:Mode$ CastWithFlash | ValidCard$ Sorcery | ValidSA$ Spell | Caster$ You | Description$ You may cast sorcery spells as though they had flash.
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigImmediateTrig | TriggerDescription$ Whenever CARDNAME, you may pay {2}. If you do, when you cast your next instant or sorcery spell this turn, copy it. You may choose new targets for the copy.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/najal_the_storm_runner.txt:6]: Should be:
Whenever CARDNAME attacks,

**Review by paulsnoops — APPROVED** [review]

**Pull request merged**